Feature home and logout buttons

Sverals opened the pull request.

Sverals committed:
Login page uses new swap logic

Login page will now swap to the landing page using new switch methods.

Sverals committed:
Swap landing page ui swap and add logout

Landing page now uses new UI swap for all buttons.
Added a logout button to go back to login page from landing page.

Sverals committed:
Implement home button on pages

All pages except unimplemented inventory page have a home button to have to the landing page.

Sverals committed:
Fix test and checkstyle

Moved constant class from viewmodel to view since it is only used in swapping.
Added documentation to clear checkstyle warnings.

Sverals committed:
Remove extra constants

Removes the page resource constants from the main page since they duplicate the ones in page resources.

kd00250 reviewed (approved):
Looks good, everything runs. Test coverage is 100%.

Sverals committed:
Fix pick storage swap

Fix the error that occurs when adding stock and swapping to the pick storage window.

Sverals committed:
Fix checkstyle

Fix the checkstyle warning after removing main usage in the add stock window.

markrockers7 reviewed (approved):
No Checkstyle issues, New buttons work as intended, 100% coverage in Model and ViewModel.
